### PR TITLE
Correct elapsed type for queue shutdown events

### PR DESCRIPTION
**Pull request by wcpines**

### CODE REVIEW

File: lib/oban/telemetry.ex [hunk 1]
```diff
@@ -312,7 +312,8 @@ defmodule Oban.Telemetry do
 
   #### Queue Shutdown Events
 
-  * `ellapsed` — the amount of time the queue waited for shutdown, in milliseconds
+  * `elapsed` — the amount of time the queue waited for shutdown, in milliseconds
+  * `ellapsed` — alias for elapsed
```
Comment (wcpines) [inline lib/oban/telemetry.ex:316]: What is the best way to preserve backwards compatibility?

Comment (sorentwo) [reply lib/oban/telemetry.ex:316]: There's no need to maintain compatibility for the log message. We can just fix the spelling.

Comment (sorentwo) [inline lib/oban/telemetry.ex:316]:
```suggestion
  * `elapsed` — the amount of time the queue waited for shutdown, in milliseconds
```

File: lib/oban/telemetry.ex [hunk 2]
```diff
@@ -458,7 +459,8 @@ defmodule Oban.Telemetry do
   def handle_event([:oban, :queue, :shutdown], measure, %{orphaned: [_ | _]} = meta, opts) do
     log(opts, fn ->
       %{
-        ellapsed: measure.ellapsed,
+        elapsed: measure.elapsed,
+        ellapsed: measure.elapsed,
```
Comment (sorentwo) [inline lib/oban/telemetry.ex:463]:
```suggestion
        elapsed: measure.elapsed,
```

**Review by sorentwo — APPROVED** [review]

**Comment by sorentwo** [comment]
Thank you!

**Pull request merged**